### ISSUE TITLE
Some doubts about Filtering Mistakes

### ISSUE THREAD

**Issue by kuaile258890** [yay-robot/yay_robot#4]
In IV.A 2), you mention 'An intuitive idea is to filter out the segments preceding corrections,' and 'You simplify this by differentiating instructions from corrections.' I wonder if this filtering involves directly removing the segments before corrections from the dataset? If it involves direct removal, wouldn't the actions caused by instructions before the corrections, which alter the robot's position and observations, subsequently affect the results on the low-level policy?
In other words, if a mistake occurs at time t, how can the last correct observation $o_{t-1}$ before the mistake reach $o_t$ after filtering, and then carry out the subsequent correction actions?"
